docs/tutorials/adding-a-support-desk-button.md

```diff
@@ -0,0 +1,243 @@
+---
+next: false
+prev: false
+title: "Integrating a Third-Party Support Desk into ApostropheCMS Admin Bar"
+detailHeading: "Tutorial"
+url: "/tutorials/adding-a-support-desk-button.html"
+content: "Learn how to add a support desk button to the ApostropheCMS admin bar, giving content editors easy access to help without leaving the CMS interface."
+tags:
+  topic: "Admin UI"
+  type: tutorial
+  effort: beginner
+---
+# Integrating a Third-Party Support Desk into ApostropheCMS Admin Bar
+
+## Why This Matters
+
+Customer support is a critical part of any website management experience. By integrating a support desk directly into the ApostropheCMS admin bar, you make it easy for content managers to get help without leaving their workflow. This integration reduces friction, improves the user experience, and ultimately leads to faster issue resolution and higher satisfaction.
+
+## Understanding Third-Party Support Desk Options
+
+Most popular third-party support desk solutions can be easily integrated with ApostropheCMS. Many create buttons that the users can click to trigger the help desk through a simple script embedded in the head of your site. However, using this approach means that you might have to worry that the end reader of your site has the option to open the help desk, which may not be desired. Adding it as a button to the admin-bar means that you can allow just logged-in users to access the support desk. But, this also means that you need to disable the automatic button addition.
+
+Common third-party support desk solutions that work well with ApostropheCMS include Zendesk Support Widget (a comprehensive customer service platform with ticketing and knowledge base), Intercom Chat (a conversational relationship platform with chat-based support), Freshdesk Messaging (customer support software with ticketing and automation), Help Scout Beacon (email-based customer support with a clean interface), and Crisp Live Chat (a lightweight customer messaging platform with chatbot capabilities).
+
+## Integration Approach
+
+### Adding the Support Button to the Admin Bar
+
+In ApostropheCMS, you can add a support desk button to the admin bar from the `init` method of any module.
+
+The code is straightforward, using the `self.apos.adminBar.add()` method to register the new button:
+
+<AposCodeBlock>
+
+```javascript
+export default {
+  init(self) {
+    // Add support desk button to the admin bar
+    self.apos.adminBar.add(
+      'support-desk',           // Unique identifier
+      'Get Support',            // Button label
+      false,                    // Available to all users with admin access
+      {
+        contextUtility: true,   // Places button in the right side utility area
+        icon: 'question-circle', // Icon for the button
+        tooltip: 'myProject:supportDesk' // Namespaced tooltip text for translation
+      }
+    );
+  }
+};
+```
+  <template v-slot:caption>
+    modules/asset/index.js
```

Comment:
I get not wanting to tell them how to create and initialize a module 50,000 times, but... maybe a quick link for the confused so people don't just create this folder exactly as shown and wonder why it does nothing without code in app.js?

Reply:
In fact: maybe just go ahead and name it `support-desk` and show activating it in `app.js`.

Reply:
Changed.